flax/experimental/nnx/nnx/module.py

```diff
@@ -292,7 +292,7 @@ def split(
   def split(
     self: M, *filters: filterlib.Filter
   ) -> tuple[State, tpe.Unpack[tuple[State, ...]], GraphDef[M]]:
-    state, graphdef = graph_utils.graph_flatten(self)
+    state, graphdef = graph_utils.graph_flatten(self)[:2]
```

Comment:
Nit: as in the test, it might be more clear to do

```
state, graphdef, _ = ...
```

instead, since you already do unpacking.